data/avgi/avgi outfits.txt

```diff
@@ -0,0 +1,608 @@
+
+
+# Engines:
+
+outfit "Magnetoplasma Drive"
+	category "Engines"
+	"cost" 30000
+	thumbnail "outfit/magnetoplasma drive"
+	"mass" 12
+	"outfit space" -12
+	"engine capacity" -12
+	"thrust" 9
+	"thrusting energy" 0.6
+	"thrusting heat" 0.3
+	"reverse thrust" 9
+	"reverse thrusting energy" 0.6
+	"reverse thrusting heat" 0.3
+	"turn" 270
+	"turning energy" 0.6
+	"turning heat" 0.3
+	"drag reduction" 1
+	"flare sprite" "effect/wanderer flare/tiny"
+		"frame rate" 5
+	"flare sound" "plasma tiny"
+	"steering flare sprite" "effect/wanderer flare/tiny"
+		"frame rate" 5
+	"steering flare sound" "plasma tiny"
+	"reverse flare sprite" "effect/wanderer flare/tiny"
+		"frame rate" 5
+	"reverse flare sound" "plasma tiny"
+	description "This compact set of magnetoplasmadynamic thrusters can propel small ships to impressive speeds over time, but is also quite energy hungry."
+	description "Designed to be clustered, this design is capable of full six-axis control."
+
+
+outfit `Z-330 "Spark" Fusion Torch`
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 660000
+	thumbnail "outfit/fusion engine small"
+	"mass" 99
+	"outfit space" -33
+	"engine capacity" -33
+	"thrust" 108
+	"thrusting energy" 9
+	"thrusting heat" 45
+	"thrusting fuel" 0.09
+	"drag reduction" 0.3
+	"flare sprite" "effect/avgi flare/fusion"
+		"frame rate" 15
+		"scale" 0.75
+	"flare sound" "zpinch"
+	"afterburner effect" "small remass injection"
+	"magnozzle rings" 3
+	description "Balance TBD"
+
+
+outfit `Z-660 "Zap" Fusion Torch`
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 1440000
+	thumbnail "outfit/fusion engine medium"
+	"mass" 198
+	"outfit space" -66
+	"engine capacity" -66
+	"thrust" 222
+	"thrusting energy" 18
+	"thrusting heat" 90
+	"thrusting fuel" 0.18
+	"drag reduction" 0.6
+	"flare sprite" "effect/avgi flare/fusion"
+		"frame rate" 15
+	"flare sound" "zpinch"
+	"afterburner effect" "medium remass injection"
+	"magnozzle rings" 6
+	description "Balance TBD"
+
+
+outfit `Z-990 "Arc" Fusion Torch`
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 2760000
+	thumbnail "outfit/fusion engine large"
+	"mass" 297
+	"outfit space" -99
+	"engine capacity" -99
+	"thrust" 360
+	"thrusting energy" 27
+	"thrusting heat" 135
+	"thrusting fuel" 0.27
+	"drag reduction" 0.9
+	"flare sprite" "effect/avgi flare/fusion"
+		"frame rate" 30
+		"scale" 2
+	"flare sound" "zpinch"
+	"afterburner effect" "large remass injection"
+	"magnozzle rings" 9
+	description "The largest of the standard fusion torches used by the Avgi"
+
+
+effect "small remass injection"
+	sprite "effect/avgi flare/afterburner"
+		"frame rate" 30
+		"scale" 0.75
+	sound "flamethrower"
+
+effect "medium remass injection"
+	sprite "effect/avgi flare/afterburner"
+		"frame rate" 30
+	sound "flamethrower"
+
+effect "large remass injection"
+	sprite "effect/avgi flare/afterburner"
+		"frame rate" 30
+		"scale" 2
+	sound "flamethrower"
+
+
+outfit "Remass Injector"
```

Comment:
This is also too strong, I think.
With these equipped, the avgi engines become more powerful per engine space than Korath afterburners, while also providing drag reduction on top of that and having the advantage of several different 'modes' (as you still have the option to run them with the normal, more fuel efficient operation of the base thruster).
Beyond that, there's also the issue that equipping these doesn't actually reduce the thrust per net space of the whole engine (when afterburning) noticeably, whereas the thrust per engine space more than doubles.
I would suggest dropping the `afterburner thrust` from 90 to 66, which makes these more of a tradeoff - still massively increasing the raw thrust per engine space, but at the cost of a lower thrust per net space.

Reply:
The thrust per net space is behind Korath afterburners, yes. My point is that by my numbers the thrust/ton with remass injectors isn't noticeably worse than with the engine firing normally, so there's not as much of a tradeoff for the remass injector relative to the engine as a whole as there perhaps should be.
![image](https://github.com/endless-sky/endless-sky/assets/18634983/52e9837d-fa35-4871-82d1-c07b3b843b29)
![image](https://github.com/endless-sky/endless-sky/assets/18634983/60d6ed61-8e88-435a-bdd9-b25cabcea379)